### PR TITLE
Typos && Minor re-phrasing

### PR DESCRIPTION
**Pull request by izzybaer**
Hi! I have a fondness for words, writing, spelling, editing and the like and came across two small typos, as well as one spot where I reworded something that I think helps the sentence to flow a bit better. Of course, these are just suggestions. I don't have any words to contribute as yet, so this is one of the best ways I know how to help.

### CODE REVIEW

File: 11ty/definitions/ok-hand.md [hunk 1]
```diff
@@ -28,6 +28,6 @@ The uncertainty created by the 4chan-ers is a critical part of their strategy. T
 
 #### Impact
 
-The people most vulnerable to the oppressive impact of White Supremacy will never know for certain if the symbol is being used in one of its non-violent manners or if it's being used in its racist violent manners. The oppressed must choose the option that protects them, which is typically erring on the side of caution.
+The people most vulnerable to the oppressive impact of White Supremacy will never know for certain if the symbol is being used in a non-violent manner or if its being used in a racist violent manner. The oppressed must choose the option that protects them, which is typically erring on the side of caution.
```
Comment (tatianamac) [inline 11ty/definitions/ok-hand.md]: stet to both corrections here
- `it's` should stay it's here as is the contraction of `it is`
- white supremacy can be used in multiple violent manners, not just one. I worry that one implies less impact.

Comment (izzybaer) [reply 11ty/definitions/ok-hand.md]: understood!

**Comment by tatianamac** [comment]
Woohoo! Thank you so much for contributing, @izzybaer ! Congrats on your first contribution. I hope it was an enjoyable experience and you'll contribute your writing talents again! 💙

**Comment by izzybaer** [comment]
thank you @tatianamac ! I definitely will :)

**Pull request merged**